spec/fixtures/authorities/linked_data/lod_lang_defaults.json

```diff
@@ -4,21 +4,21 @@
     "url": {
       "@context": "http://www.w3.org/ns/hydra/context.jsonld",
       "@type":    "IriTemplate",
-      "template": "http://localhost/test_default/term/{term_id}",
+      "template": "{term_uri}",
```

Comment:
This could have been updated to use the workcat request API instead of the bogus `localhost/test_default`, but it is more flexible to simply use `{term_uri}`.  This allows this config to be used with multiple authorities for testing.